### ISSUE TITLE
twisted.conch.ssh.transport should be ported to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8638]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)| @einarfd reported|
|-|-|
|Trac ID|trac#8638|
|Type|enhancement|
|Created|2016-07-20 14:11:29Z|

The twisted.conch.ssh.transport module should be ported to Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8638 8638
type__enhancement enhancement
reporter__einarfd einarfd
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469023889925155 1469023889925155
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)|@einarfd set owner to @einarfd|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)<a name="note_2"></a>|@einarfd removed owner|
|-|-|

Added pull request https://github.com/twisted/twisted/pull/367 to fix this issue.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset e94222cb3a388b4dc45e15ba44c42cf0d0e84106
```
#!CommitTicketReference repository="" revision="e94222cb3a388b4dc45e15ba44c42cf0d0e84106"
Merge pull request #376 from twisted/8638-einarfd-ssh-transport

Author: einarfd
Reviewer: rodrigc
Fixes: #8638

Port twisted.conch.ssh.transport to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
